README.md

```diff
@@ -1 +1,104 @@
-# AppModelv2-NativeClient-DotNet
+---
+services: active-directory
+platforms: dotnet
+author: jmprieur
+level: 200
+client: Windows Desktop WPF
+service: ASP.NET Web API
+endpoint: AAD V2
+---
+
+# Calling an ASP.NET Web API protected by the Azure AD V2 endpoint from an Windows Desktop (WPF) application
+
+## About this sample
+
+### Scenario
+
+You expose a Web API and you want to protect it so that only authenticated user can access it. This sample shows how to expose a ASP.NET Web API so it can accept tokens issued by personal accounts (including outlook.com, live.com, and others) as well as work and school accounts from any company or organization that has integrated with Azure Active Directory.
+
+The sample also include a Windows Desktop application (WPF) that demonstrate how you can request an access token to access a Web APIs.
+
+## How to run this sample
+
+> Pre-requisites: This sample requires Visual Studio 2015 Update 3 or Visual Studio 2017. Don’t have it? Download [Visual Studio 2017 for free](https://www.visualstudio.com/downloads/).
+
+### Step 1: Download or clone this sample
+
+You can clone this sample from your shell or command line:
+
+  ```console
+  git clone https://github.com/AzureADQuickStarts/AppModelv2-NativeClient-DotNet.git
+  ```
+
+### Step 2: Register your Web API - *TodoListService* in the *Application registration portal*
+
+1. Sign in to the [Application registration portal](https://apps.dev.microsoft.com/portal/register-app) either using a personal Microsoft account (live.com or hotmail.com) or work or school account.
+1. Give a name to your Application, such as `AppModelv2-NativeClient-DotNet-TodoListService`. Make sure that the *Guided Setup* option is **Unchecked** then press **Create**. The portal will assign your app a globally unique *Application ID* that you'll use later in your code.
+1. Click **Add Platform**, and select **Web API**
+1. Click **Save**
+
+> Note: When you add a *Web API* the Application registration portal, it adds a pre-defined App Id URI and Scope, using the format *api://{Application Id}/{Scope Name}* named **access_as_user** (you can review it by clicking 'Edit' button). This sample code uses this default scope.
+
+### Step 3: Configure your *TodoListService* project to match the Web API you just registered
```

Comment:
Rather: 
### Step 3: Configure your *TodoListService* and *TodoListClient* projects to match the Web API you just registered
as Step 3.1 is about the todolist client?

Alternatively we might want to move 3.1 under 5 ?